source/whats-new.txt

```diff
@@ -15,21 +15,42 @@ What's New
    :values: reference
 
 .. meta::
-   :keywords: update, new feature, deprecation, upgrade, driver v4.7, driver v4.8, driver v4.9
+   :keywords: update, new feature, deprecation, upgrade, driver v4.7, driver v4.8, driver v4.9, driver v4.10
 
 Learn what's new in:
 
+* :ref:`Version 4.10 <version-4.10>`
 * :ref:`Version 4.9 <version-4.9>`
 * :ref:`Version 4.8 <version-4.8>`
 * :ref:`Version 4.7 <version-4.7>`
 
 .. _upcoming-breaking-changes:
 
+.. _version-4.10:
+
+What's New in 4.10
+------------------
+
+The {+driver-short+} v4.10 release includes the following new features:
+
+- Adds provisional support for a new binary BSON subtype (9), which can be used
+  for efficient storage and retrieval of :atlas:`vectors
+  </atlas-vector-search/vector-search-overview/#std-term-vector>` by using the
+  ``bson.binary.Binary.from_vector()`` and ``bson.binary.Binary.as_vector()``
+  methods. Support for BSON subtype 9 is in beta and is subject to change before the
+  generally available release.
+- Adds ``"c"`` to the ``driver.name`` client metadata field when an application uses the C extension.
+- Fixes a bug that could cause ``AsyncMongoClient`` to deadlock.
+- Fixes a bug that causes Windows to fail when importing {+driver-short+} if
```

Comment:
We can remove this, it turned out to be an incomplete fix, and was a problem with with how we were running a test in EVG.

Reply:
Removed